translations/messages+intl-icu.de.yml

```diff
@@ -1634,6 +1634,7 @@ feedbackto: "Rückmeldung an"
 field: "Feld"
 field.empty: "Feld leeren"
 field.required: "Pflichtfeld"
+field.selectionRequired: "Mindestens ein Feld muss ausgewählt werden"
```

Comment:
Nitpick: Even if the other keys around here miss any punctuation, this is a sentence and needs a dot at the end - at least that's what i feel.